examples/c/src/sum.c

```diff
@@ -0,0 +1,14 @@
+#include <stdio.h>
+int main()
+{
+  int x, y, z;
+
+  printf("Enter two numbers to add: \n");
+  scanf("%d %d", &x, &y);
+
+  z = x + y;
+
+  printf("Sum of the numbers : %d\n", z);
+
+  return 0;
+}
```

Comment:
This seems unrelated?

Reply:
@vladaionescu I have added 'prime_check.rb' file to the 'ruby' directory. This is the C file that you are viewing.

Reply:
Right.. the question is why is this in this PR